ocamltest/Makefile

```diff
@@ -33,8 +33,16 @@ else
 endif
 
 ifeq "$(filter $(UNIXLIB),$(OTHERLIBRARIES))" ""
+  ocamltest_unix := dummy
+	unix_name :=
+	unix_path :=
   unix := None
```

Comment:
Is this variable `$(unix)` used anywhere?

Reply:
Yes - it's `Ocamltest_config.libunix` (see `ocamltest_config.ml.in` line 32 and `Makefile` line 262)